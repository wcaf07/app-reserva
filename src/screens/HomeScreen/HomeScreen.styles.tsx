import styled from 'styled-components/native';
import colors from '@tokens/colors';

const Container = styled.SafeAreaView`
  flex: 1;
  justify-content: flex-end;
  background-color: ${colors.white};
`;

const ImageTitle = styled.Image`
  width: 200px;
  height: 200px;
  align-self: center;
  margin-bottom: 100px;
`;

export {Container, ImageTitle};
