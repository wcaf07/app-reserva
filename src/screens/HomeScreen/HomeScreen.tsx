import React from 'react';
import { Text } from 'react-native';

const HomeScreen: React.FC = () => {
  return (
    <Text>Home Screen</Text>
  )
}

export default HomeScreen;
