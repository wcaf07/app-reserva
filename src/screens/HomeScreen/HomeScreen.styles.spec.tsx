import React from 'react';
import {render} from '@testing-library/react-native';
import {Container, ImageTitle} from './HomeScreen.styles';

describe('HomeScreen.styles', () => {
  test('THEN ImageTitle SHOULD match with snapshot when is show', () => {
    const component = render(<ImageTitle />).toJSON();

    expect(component).toMatchSnapshot();
  });
  test('THEN Container SHOULD match with snapshot when is show', () => {
    const component = render(<Container />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
