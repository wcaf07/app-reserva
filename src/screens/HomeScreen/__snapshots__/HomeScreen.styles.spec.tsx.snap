// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen.styles THEN Container SHOULD match with snapshot when is show 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-end",
      },
    ]
  }
/>
`;

exports[`HomeScreen.styles THEN ImageTitle SHOULD match with snapshot when is show 1`] = `
<Image
  style={
    Array [
      Object {
        "alignSelf": "center",
        "height": 200,
        "marginBottom": 100,
        "width": 200,
      },
    ]
  }
/>
`;
